### PR TITLE
Fluff items worshop....5? Sequels are hard.

### PR DESCRIPTION
**Pull request by JoanRisu**
Lots changes. Mostly involves getting the charges right for Aro's gun and my Dominator, attempts to fix Molener's items, little typos fixes here and there, and balancing for armor items. Also changes some item sprites.

### CODE REVIEW

File: code/modules/vore/fluffstuff/custom_permits_vr.dm [hunk 1]
```diff
@@ -23,16 +23,6 @@
 	This license expires on DD/Mon/YYYY and must be renewed by CentCom prior to this date."}
 */
 
-// bwoincognito:Tasald Corlethian
-/obj/item/fluff/permit/tasald_corlethian
```
Comment (SpadesNeil) [inline code/modules/vore/fluffstuff/custom_permits_vr.dm]: This doesn't need to be deleted. Why is it being deleted? It's not even expired yet.

Comment (JoanRisu) [inline code/modules/vore/fluffstuff/custom_permits_vr.dm]: Because its a duplicate from what I saw.

Comment (JoanRisu) [inline code/modules/vore/fluffstuff/custom_permits_vr.dm]: Yup, this is a duplicate of another entry.

**Comment by JoanRisu** [comment]
Also, make sure no one else is editing the custom items and guns .dmi files since I worked on those on this one.

File: code/modules/vore/fluffstuff/custom_permits_vr.dm (hunk 1 above)
Comment (SpadesNeil) [inline code/modules/vore/fluffstuff/custom_permits_vr.dm]: Oh ok.

**Pull request merged**